demo/src/test/java/amazon/SearchFieldTest.java

```diff
@@ -0,0 +1,137 @@
+package amazon;
+
+import java.time.Duration;
+import java.util.List;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class SearchFieldTest {
+   protected WebDriver driver;
+   protected String searchData = "laptop"; //default
```

Comment:
Not critical, but some comments are redundant

Reply:
this for my teammates)